topics/genome-annotation/tutorials/apollo-euk/tutorial.md

```diff
@@ -33,6 +33,8 @@ contributions:
   funding:
     - gallantries
     - eurosciencegateway
+    - elixir-europe
+    - ifb
```

Comment:
not sure about these here?  that work was more or less strictly gallantries funded, we weren't paid by e.g. the elixir galaxy IS or anything?